packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerCalendar.tsx

```diff
@@ -37,7 +37,6 @@ export const getStyles = (theme: GrafanaTheme2, isReversed = false) => {
     modal: css`
       position: fixed;
       top: 20%;
-      left: 25%;
```

Comment:
Ideally we would want this modal to pop up next to the button that called it, but this at least will have it show consistently in the middle of the window. I feel like the left position was put here for a reason, however, so let me know what my options might be short of a heavy refactor of the calendar modal.